crates/next-api/src/dynamic_imports.rs

```diff
@@ -103,6 +103,23 @@ pub(crate) async fn collect_evaluated_chunk_group(
     .await
 }
 
+#[turbo_tasks::value(shared)]
+pub struct NextDynamicImportsResult {
+    pub client_dynamic_imports: IndexMap<Vc<Box<dyn Module>>, DynamicImportedModules>,
+    pub visited_modules: Vc<VisitedDynamicImportModules>,
+}
+
+#[turbo_tasks::value(shared)]
+pub struct VisitedDynamicImportModules(HashSet<NextDynamicVisitEntry>);
+
+#[turbo_tasks::value_impl]
+impl VisitedDynamicImportModules {
+    #[turbo_tasks::function]
+    pub fn empty() -> Vc<Self> {
+        VisitedDynamicImportModules(Default::default()).cell()
+    }
+}
```

Comment:
Could be done with `ValueDefault` to have `Vc::<VisitedDynamicImportModules>::default()`

Reply:
`empty()` seems to be somewhat of a convention though, if you search for `empty() -> Vc<Self>` in the codebase. Maybe we should unify that at some point.